llpc/util/llpcElfWriter.cpp

```diff
@@ -1118,6 +1193,142 @@ void ElfWriter<Elf>::Reinitialize()
     m_map[NoteName] = m_noteSecIdx;
 }
 
+uint32_t GetResourceOffset(uint32_t descSet, uint32_t binding, ResourceMappingNodeType type, const ResourceMappingNode* userResources, uint32_t
+    resourceCount);
+
+struct RelocationEntry
+{
+    uint32_t offset;
+    const char* name;
+};
```

Comment:
Should be in an anonymous namespace. Also, this should contain the relocation type.

Reply:
Done.